src/System.Runtime/samples/DateTime-formatting/DateTime-formatting.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using System.Globalization;
+
+public class Program
+{
+	public static void Main()
+	{
+		// Demonstrates common uses of formatting the DateTime struct.
+		// Doc: https://msdn.microsoft.com/library/az4se3k1.aspx
+		
+		var date = DateTime.Now;
```

Comment:
Could we change ```date``` here to be ```localData```, or something that highlights better what ```Now``` returns?